packages/client/hooks/useSoundEffect.ts

```diff
@@ -0,0 +1,46 @@
+import {useEffect, useRef} from 'react'
+
+type Props = {startTime: Date | string} | {endTime: Date | string}
```

Comment:
+1 a dev could add both the `startTime` and the `endTime` to this hook which would prevent it from working. 

Could change the type to prevent that from happening:
```
type StartTimeProps = {startTime: Date | string; endTime?: never}
type EndTimeProps = {endTime: Date | string; startTime?: never}
type Props = StartTimeProps | EndTimeProps
```